src/QirRuntime/build-qir-runtime.ps1

```diff
@@ -71,7 +71,7 @@ elseif (($IsWindows) -or ((Test-Path Env:AGENT_OS) -and ($Env:AGENT_OS.StartsWit
     Write-Host "##vso[task.logissue type=error;]Failed to identify the OS. Will use default CXX compiler"
 }
 
-$qirRuntimeBuildFolder = (Join-Path $PSScriptRoot "build\$Env:BUILD_CONFIGURATION")
+$qirRuntimeBuildFolder = (Join-Path $PSScriptRoot build $Env:BUILD_CONFIGURATION)
```

Comment:
I think this makes sense, but as a portability note, this is incompatible with PowerShell 5.1 (deprecated, but still the default on fresh Windows 10 installs). It's less elegant, but for compatibility with 5.1 as to @bettinaheim's recent point, can we modify this as follows?

```suggestion
$qirRuntimeBuildFolder = (Join-Path $PSScriptRoot (Join-Path build $Env:BUILD_CONFIGURATION))
```

Reply:
(see for example the [prerequisites section](https://github.com/microsoft/qsharp-runtime/blob/main/src/QirRuntime/README.md#prerequisites) of the readme in this folder with @kuzminrobin recently updated to reflect the PowerShell 7 requirement)

Reply:
Ah, please disregard, then; I had gotten my wires crossed on 5.1 vs 7.0 requirements, then!